modules/ossm-about-concepts-kiali.adoc

```diff
@@ -0,0 +1,71 @@
+
+Prometheus:: A dedicated Prometheus instance optional. When Istio telemetry is enabled, metrics data are stored in Prometheus. Kiali uses this Prometheus data to determine the mesh topology, display metrics, calculate health, show possible problems, and so on. Kiali communicates directly with Prometheus and assumes the data schema used by Istio Telemetry. Prometheus is an Istio dependency and a hard dependency for Kiali, and many of Kiali's features will not work without Prometheus.
+
+{ocp-product-title} API:: Kiali uses {ocp-product-title} API to fetch and resolve service mesh configurations. Kiali queries the cluster API to retrieve, for example, definitions for namespaces, services, deployments, pods, and other entities. Kiali also makes queries to resolve relationships between the different cluster entities. The cluster API is also queried to retrieve Istio configurations like virtual services, destination rules, route rules, gateways, quotas, and so on.
+
+Tracing:: Tracing is optional, but when you install {DTProductName} and Kiali is configured, the Kiali console includes a tab to display distributed tracing data, and tracing integration on the graph itself. Note that tracing data will not be available if you disable Istio’s distributed tracing feature. Also note that the user must have access to the namespace where the user needs to see tracing data.
+
+Grafana:: Grafana is optional. When available, the metrics pages of Kiali display links to direct the user to the same metric in Grafana. Note that Grafana is not supported as part of {ocp-product-title} or {SMProduct}.
+
+[id="kiali-features_{context}"]
+== Kiali features
+
+The Kiali console is integrated with {SMProduct} and provides the following capabilities:
+
+Health:: Quickly identify issues with applications, services, or workloads.
+
+Topology:: Visualize how your applications, services, or workloads communicate via the Kiali graph.
+
+Metrics::  Predefined metrics dashboards let you chart service mesh and application performance for Go, Node.js. Quarkus, Spring Boot, Thorntail and Vert.x. You can also create your own custom dashboards.
+
+Tracing:: Integration with {TempoName} lets you follow the path of a request through various microservices that make up an application.
+
+Validations:: Perform advanced validations on the most common Istio objects (Destination Rules, Service Entries, Virtual Services, and so on).
+
+Configuration:: Optional ability to create, update and delete Istio routing configuration using wizards or directly in the YAML editor in the Kiali Console.
+
+[id="openshift-service-mesh-console-plugin_{context}"]
+== {SMPlugin}
+
+The {SMPlugin} is an {ocp-product-title} plugin for {SMProductName}. It integrates much of the {KialiProduct} interface into the OpenShift Console, injecting both a *Service Mesh* main menu option with dedicated screens, as well as integrating “Service Mesh” tabs throughout console.
```

Comment:
Not sure this is right. I am just unfamiliar with using "" around UI elements. Shrug...
```suggestion
The {SMPlugin} is an {ocp-product-title} plugin for {SMProductName}. It integrates much of the {KialiProduct} interface into the OpenShift Console, injecting both a *Service Mesh* main menu option with dedicated screens, as well as integrating *Service Mesh* tabs throughout console.
```